fluent-dom/src/localization.js

```diff
@@ -1,28 +1,44 @@
 /* eslint no-console: ["error", { allow: ["warn", "error"] }] */
 /* global console */
 
+import CachedIterable from '../../fluent/src/cached_iterable';
```

Comment:
Why can't we leave `CachedIterable` out of this file and keep it as a responsibility of the glue code?